crates/sargon/src/wrapped_radix_engine_toolkit/low_level/v2/transaction_manifest_v2/child_intents.rs

```diff
@@ -2,46 +2,58 @@ use crate::prelude::*;
 
 /// Represents a collection of child subintents.
 ///
-/// This struct is used to manage a list of `ChildSubintent` instances, providing
+/// This struct is used to manage a list of `ChildSubintentSpecifier` instances, providing
 /// methods for creation, conversion, and sample values for testing purposes.
 ///
 /// # Fields
-/// - `children`: A vector of `ChildSubintent` instances.
+/// - `children`: A vector of `ChildSubintentSpecifier` instances.
 #[derive(Clone, Debug, PartialEq, Eq)]
-pub struct ChildIntents {
-    pub children: Vec<ChildSubintent>,
+pub struct ChildSubintentSpecifiers {
```

Comment:
@matiasbzurovski we should rename this file to `
child_subintent_specifiers` , I leave that to you